Makefile.in

```diff
@@ -111,12 +111,14 @@ umu-launcher-dist-install:
 
 umu-launcher-install: umu-launcher-dist-install umu-launcher-bin-install
 
+
```

Comment:
Unrelated blank line?
```suggestion
```

Reply:
intended change for easier visual grouping